### DISCUSSION TITLE
중급프로젝트 3주차 팀회고

### DISCUSSION THREAD

**Discussion by yeji0512** [codeit-bootcamp-frontend/2-Weekly-Mission#431] [회고]
중급 프로젝트 3주차 팀 회고를 하고([회고 방식 상세보기](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/discussions/429#discussion-6130547)), 이전처럼 댓글 남겨주세요. 

설 연휴 관계로, 이번 팀 회고는 선택입니다. 
회고가 필요하다고 느끼신다면, 이번주도 회고도 꼭 해보세요 😎 

설날 연휴 잘 보내시고, 남은 프로젝트도 화이팅🔥🍀☘️

![image](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/144666341/5b88de00-c7d2-48ff-a4a6-3c7d16a0fc50)

**Comment by eonpain** [comment]
# 5팀

## KEEP
- 명절에 껴서 위워크에서 직접 소통은 못하지만, 팀원 3명 전부 디스코드에 익숙해서 그런지 꾸준히 소통함
- 코드 리뷰, vscode live share를 통해서 문제해결이 잘 이루어짐

## PROBLEM

## TRY
- 사용자의 입장에서 우리가 만든 프로젝트를 사용하는데 불편함은 없는지 체크하기
- 기능을 구현했더라도 최대한 리팩토링 해보기
